### PR TITLE
add webhookify.io

### PR DESCRIPTION
**Pull request by Gertiozuni**
* [ ] This is Software as a Service not self hosted
 * [x] It has a free tier not just a free trial
 * [x] Pricing information is clearly visible without signup or phone calls
 * [x] The submission mentions what is free
 * [x] The submission is not already present in the list
 * [x] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -374,6 +374,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
   * [OpenWeb Ninja](https://www.openwebninja.com/) - Extremely comprehensive real-time SERP and public data APIs: Google Search, Shopping, Jobs, Images, Lens, News, Google Maps Businesses / Places, Reviews, Photos, Website Emails and Social Contacts Scraper, Amazon, Yelp and more. All APIs include a free tier with 100 to 200 free monthly requests.
   * [Tavily AI](https://tavily.com/) - API for online serach and rapid insights and comprehensive research, with the capability of organization of research results. 1000 request/month for the Free tier with No credit card required.
   * [Zuplo](https://zuplo.com) - Free API Management platform to design, build, and deploy APIs to the Edge. Add Authentication, Rate Limiting, API Documentation, and Monetization in minutes. OpenAPI-native and fully-programmable with web standard apis & Typescript.
+  * [Webhookify](https://webhookify.io/download) - Free desktop app in macos/windows/linux to easily test, debug and integrate webhooks like never before. Generate URLs, Capture Requests, Replay, Repeat. No login/sign up required.
```
Comment (ripienaar) [inline README.md:377]: Sorry this list if for SaaS, only exception really is for generic IDEs

Comment (Gertiozuni) [reply README.md:377]: Do i need to put on any other section, or my devtool doesn't qualify for this list at all?

Comment (ripienaar) [reply README.md:377]: The list is for SaaS.

Comment (Gertiozuni) [reply README.md:377]: Got it, thank you

**Pull request closed without merging**